templates/Search/results.html.twig

```diff
@@ -37,7 +37,7 @@
               <li>{{ coaster.item.materialType }}</li>
             {% endif %}
             {% if coaster.distance is defined %}
-              <li>{{ 'search_index.km_away'|trans({'%num%': coaster.distance|number_format(0)}) }}</li>
+              <li>{{ 'search_index.km_away'|trans({'%num%': unities.isImperial(app.request.locale) ? (coaster.distance / 1.609)|round(0) : coaster.distance|round(0), '%unity%': unities.isImperial(app.request.locale) ? ' mil' : ' km' }) }}</li>
```

Comment:
you are duplicating the function logic here, introduce instead a parameter to round it inside the m_or_f function

Reply:
The main problem was the translation I think. I need to display a different sentence depending on the language ("8 mi away" in english, "à 8 km" in french for example), so I updated the convert functions.

They now return an array with the value and the units, the two types of data can be used differently.